internal/db/db.go

```diff
@@ -25,7 +25,7 @@ import (
 
 // Open opens the dqlite database and loads the schema.
 // Returns true if we need to wait for other nodes to catch up to our version.
-func (db *DB) Open(ext extensions.Extensions, bootstrap bool, project string) error {
+func (db *DB) Open(ext extensions.Extensions, bootstrap bool, project string, memberName string) error {
```

Comment:
Instead of specifying the name here, can we pass the `daemon.Name` function to `NewDB` like we do for the certificates? That way all the daemon configuration is passed in at once.

Reply:
I started out with the name itself as a field of `DB`; passing the function is a far more elegant solution...